mlir/lib/Dialect/Tosa/Transforms/TosaValidation.cpp

```diff
@@ -401,27 +406,39 @@ struct TosaValidation : public tosa::impl::TosaValidationBase<TosaValidation> {
 
     if (!profile.empty()) {
       for (std::string &prof : profile) {
-        auto profSymbol = symbolizeTosaProfileEnum(prof);
+        auto profSymbol = symbolizeProfile(prof);
         if (profSymbol) {
-          enabled_profiles.push_back(profSymbol.value());
+          targetEnv.addProfile(profSymbol.value());
+        } else {
+          llvm::errs() << "warning: unknown profile name passed in, supported "
+                          "profile are bi and mi\n";
+        }
```

Comment:
Is it worth `signalPassFailure()` when an invalid option is passed in? Just thinking of the case someone mistypes a name and subsequently spammed by lots of op errors

Reply:
Yes, fair enough. I forgot to add it back.

Reply:
Apologies, I think we also need to return to avoid running the validation

Reply:
Agree. The term `warning` is removed.